src/main/resources/reference.conf

```diff
@@ -36,6 +36,7 @@ conseil {
   cache-ttl: 15 minutes
   max-query-result-size: 100000
   high-cardinality-limit: 100
+  startup-deadline: 1 minute
```

Comment:
I would recommend a longer deadline. We use this functionality when multiple containers are being brought up together. It's possible the database container might take longer to boot up. How about five minutes?

Reply:
No problem